### PR TITLE
chore: add volunteer PriteshKiri

### PR DESCRIPTION
**Pull request by PriteshKiri**

### CODE REVIEW

File: data/users/PriteshKiri.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Pritesh Kiri",
+    "bio": "Fronend Dev",
+    "image": "https://avatars.githubusercontent.com/u/70970530?v=4",
```
Comment (codewithvoid) [inline data/users/PriteshKiri.json:4]: Use your image url :)

**Comment by codewithvoid** [comment]
Thanks for volunteering @PriteshKiri . Added some comments.

File: data/users/PriteshKiri.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["CSS", "javascript","react","gatsby"]
```
[the remark this answers is not in the document]
Comment (PriteshKiri) [reply data/users/PriteshKiri.json:8]: Alright!

File: data/users/PriteshKiri.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Pritesh Kiri",
+    "bio": "Fronend Dev",
```
Comment (codewithvoid) [inline data/users/PriteshKiri.json:3]: typo fix

File: data/users/PriteshKiri.json (hunk 1 above)
Comment (PriteshKiri) [reply data/users/PriteshKiri.json]: Just edited! my bad :/

File: data/users/PriteshKiri.json (hunk 3 above)
Comment (PriteshKiri) [reply data/users/PriteshKiri.json]: done! :|

Comment (PriteshKiri) [reply data/users/PriteshKiri.json]: done! :|

**Comment by codewithvoid** [comment]
Thanks for making quick changes. Welcome!

**Pull request merged**